src/library/coap.hpp

```diff
@@ -689,12 +689,9 @@ class Coap
             return *mContainer.begin();
         }
 
-        // Clear all requests and stop retransmission timer.
-        void Clear();
-
-        // Try restart the retransmit timer if it is not running and there is
-        // pending requests.
-        void TryRetartTimer();
+        // Try starting the retransmit timer if it is not running
+        // and there is pending requests.
+        void TryStartTimer();
```

Comment:
```suggestion
        void UpdateTimer();
```

nit: `UpdateTimer` or  `ResetTimer` might be a better name. It can also stop timer if it's no longer needed.